src/common/utxobased/engine/makeUtxoEngineState.ts

```diff
@@ -881,7 +891,7 @@ const internalDeriveScriptAddress = async (
   return { address, scriptPubkey, redeemScript }
 }
 
-interface GetFreshAddressArgs extends FormatArgs { }
+interface GetFreshAddressArgs extends FormatArgs {}
```

Comment:
I know it's not part of the PR but on line 1234, why we are not using a const here:
``` const [tx] = await processor.fetchTransactions({ txId: utxo.txid })```

Reply:
https://github.com/EdgeApp/edge-currency-plugins/pull/198